### PR TITLE
MM-1762 - Search improvements with some other minor Ui changes

### PR DESCRIPTION
**Pull request by asaadmahmoodspin**
This is in for 0.6.0. Please have a look.

### CODE REVIEW

File: web/react/components/sidebar_header.jsx [hunk 1]
```diff
@@ -75,7 +75,7 @@ var NavbarDropdown = React.createClass({
 
         return (
             <ul className="nav navbar-nav navbar-right">
-                <li className="dropdown">
+                <li className="dropdown" id="myDropdown">
```
Comment (nickago) [inline web/react/components/sidebar_header.jsx]: Is this going to be used for something in the future?

**Comment by nickago** [comment]
Code looks good, just one style question

File: web/react/components/sidebar_header.jsx (hunk 1 above)
Comment (asaadmahmoodspin) [inline web/react/components/sidebar_header.jsx]: No, I guess I forgot to remove it. Will remove it right now.

**Comment by nickago** [comment]
Alright, I give this my +1, just waiting on a seconder

**Comment by jwilander** [comment]
+1

**Pull request merged**